src/lib.rs

```diff
@@ -126,11 +127,11 @@ impl Importer for CheckImporter {
         let TypecheckValue { expr, typ } =
             try!(macro_value.typecheck(compiler, vm, module_name, input));
 
-        let lines = source::Lines::new(input);
+        let lines = source::Lines::new(input.as_bytes().iter().cloned());
```

Comment:
I am not sure if this is correct. It compiles, but do we need to clone, or is there a better way to approach this?

Reply:
No that is fine (it got changed from taking `&str` to support ropes as a source code representation, but then I never ended up needing ropes anyway... I may change that in the future however).